common/djangoapps/student/tests/tests.py

```diff
@@ -404,6 +414,91 @@ def test_refundable_when_certificate_exists(self):
         self.assertFalse(enrollment.refundable())
 
     @unittest.skipUnless(settings.ROOT_URLCONF == 'lms.urls', 'Test only valid in lms')
+    def test_refundable_when_refund_window(self):
+        CourseModeFactory.create(
+            course_id=self.course.id,
+            mode_slug='verified',
+            mode_display_name='Verified',
+            expiration_datetime=datetime.now(pytz.UTC) + timedelta(days=1)
+        )
+
+        enrollment = CourseEnrollment.enroll(self.user, self.course.id, mode='verified')
+
+        self.assertTrue(enrollment.refundable())
+
+        with patch('student.models.CourseEnrollment._refund_window_end_date') as window:
+            window.return_value = datetime.now() - timedelta(days=1)
```

Comment:
Add comments explaining what you're testing. It may seem obvious to you now; but, it won't be so obvious in a few weeks.